src/bbs2gh/Services/ReposCsvGeneratorService.cs

```diff
@@ -35,12 +33,15 @@ public virtual async Task<string> Generate(string bbsServerUrl, string bbsUserna
             {
                 foreach (var repo in await inspector.GetRepos(projectKey))
                 {
-                    var url = $"{bbsServerUrl.TrimEnd('/')}/projects/{projectKey}/repos/{repo.Slug}";
-                    var lastCommitDate = await inspector.GetLastCommitDate(projectKey, repo.Slug);
-                    var (repoSize, attachmentsSize) = await inspector.GetRepositoryAndAttachmentsSize(projectKey, repo.Slug, bbsUsername, bbsPassword);
+                    var url = $"{bbsServerUrl.TrimEnd('/')}/projects/{projectKey}/repos/{Uri.EscapeDataString(repo.Slug)}";
```

Comment:
you should probably escape projectKey also. In general whenever we are mashing things together to form a URL, all of the individual pieces of dynamic data should be run through that escaping function.
```suggestion
                    var url = $"{bbsServerUrl.TrimEnd('/')}/projects/{Uri.EscapeDataString(projectKey)}/repos/{Uri.EscapeDataString(repo.Slug)}";
```

Reply:
Done [7303ce7](https://github.com/github/gh-gei/pull/1015/commits/7303ce7722317a9226e6e3e10d7fc502c9da989f)